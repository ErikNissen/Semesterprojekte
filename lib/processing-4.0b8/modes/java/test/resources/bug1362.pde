version https://git-lfs.github.com/spec/v1
oid sha256:bf527f069e0a67697fa4e1d3b6d25ca4d96a2bf11f08c743c023030ff14ebda5
size 35
