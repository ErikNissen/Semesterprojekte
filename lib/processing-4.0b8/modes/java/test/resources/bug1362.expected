version https://git-lfs.github.com/spec/v1
oid sha256:c3840e55310722624562439453ca9e63a910ca802b7c15bdfe635928e964ca4d
size 729
